fix power control for multi-host configuration

aasboddu opened the pull request:
This changeset disables the check for multiple TPMEnable objects since the condition blocks power control operations when multi-host support is enabled.

ojayanth commented:
@aasboddu  this is not fix power-controil problem , Here just disabling TPM feature for multihost enabled system. you can rephase the PR accordingly.

ojayanth commented on redfish-core/lib/systems.hpp:
```diff
@@ -1415,17 +1415,6 @@ inline void getTrustedModuleRequiredToBoot(
             return;
         }
 
-        /* When there is more than one TPMEnable object... */
```
Change the subject to align to your chnages. multi-host: disable xxxx

ojayanth commented on redfish-core/lib/systems.hpp:
```diff
@@ -1415,17 +1415,6 @@ inline void getTrustedModuleRequiredToBoot(
         }
 
-        /* When there is more than one TPMEnable object... */
-        if (subtree.size() > 1)
```
instead of commenting everything just disble internal error  return and update log message syaying  required to enable fduring multihost TPM enbalemnet.

aasboddu committed:
fix power control for multi-host configuration

This commit disables the check for multiple TPMEnable
objects. This condition check breaks the power control
operations from being performed when multi-host support
is enabled.

Signed-off-by: Aashray Boddu <Aashray.Boddu@amd.com>
